lib/sessionManager.ts

```
@@ -0,0 +1,290 @@
+   */
+  static isSuspiciousSession(session: DetailedSession, currentSession: DetailedSession | null): boolean {
+    if (!currentSession || session.current) {
+      return false;
+    }
+    
+    return session.countryCode !== currentSession.countryCode && 
+           session.countryCode !== 'unknown' && 
+           currentSession.countryCode !== 'unknown';
+  }
+}
+
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)